posts/README.md

```diff
@@ -0,0 +1,84 @@
+# large-events/posts
+
+Posts service as part of large-events project for Knative proof of portability.
```

Comment:
I'd suggest including a 1-2 line description of what the service does.

Reply:
Done https://github.com/knative-portability/large-events/pull/3/commits/d9fe2d021b3d3c3214f670edee1536fc5cebad3a